"markdown-it-wikilinks" dependency is not packed

## To Reproduce

1. Install [version 0.0.20](https://github.com/kortina/vscode-markdown-notes/releases/tag/v0.0.20).
2. Restart VS Code, and open a Markdown document.
3. See **Developer Tools** > **Console**:

   ```log
   Error: Cannot find module '@thomaskoppelaar/markdown-it-wikilinks'
   ```

## Behind the scene

Unzip `vscode-markdown-notes-0.0.20.vsix`.

In `out/MarkdownRenderingPlugin.js`, at line 50:

```js
return require('@thomaskoppelaar/markdown-it-wikilinks')({
```

But `node_modules/@thomaskoppelaar/markdown-it-wikilinks/` does not exist, so this statement will **crash**.

The reason is as follows.

By default, vsce packs **production** dependencies, unless you exclude them.

However, `markdown-it-wikilinks` is not there:

```shell
$ npm ls --production --parseable

node_modules/github-slugger
node_modules/github-slugger/node_modules/emoji-regex
```

It is under dev dependencies currently:
<https://github.com/kortina/vscode-markdown-notes/blob/431084cdae86891ab55cf01c8cb9983fcb377384/package.json#L219>

## Versions

- OS: Windows 10 Pro, Version 1909, Build 10.0.18363.1082, 64-bit
- VS Code Version: 1.50.0
- Extension Version: 0.0.20

## Possible solution

Move `@thomaskoppelaar/markdown-it-wikilinks` to production dependencies:

```shell
npm install --save-prod @thomaskoppelaar/markdown-it-wikilinks
```

--- a/package.json
+++ b/package.json
@@ -188,6 +188,7 @@
     "watch": "tsc -watch -p ./"
   },
   "dependencies": {
+    "@thomaskoppelaar/markdown-it-wikilinks": "^1.3.0",
     "github-slugger": "^1.3.0"
   },
   "devDependencies": {
@@ -216,7 +217,6 @@
     "unified": "^9.0.0",
     "unist-util-find": "^1.0.1",
     "unist-util-visit": "^2.0.2",
-    "@thomaskoppelaar/markdown-it-wikilinks": "1.3.0",
     "vsce": "^1.75.0",
     "vscode-test": "^1.3.0"
   }

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "vscode-markdown-notes",
   "displayName": "Markdown Notes",
   "description": "Navigate notes with [[wiki-links]], backlinks, and #tags (like Bear, Roam, etc). Automatically create notes from new inline [[wiki-links]]. Use Peek Definition to preview linked notes.",
-  "version": "0.0.20",
+  "version": "0.0.21",
   "publisher": "kortina",
   "repository": {
     "url": "https://github.com/kortina/vscode-markdown-notes.git",

